apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/question/request/CreateQuestionRequest.java

```diff
@@ -35,4 +38,100 @@ public record MessagingInfo(
                         String hl7DataType) {
         }
 
+        public enum CodedQuestionType {
+                SINGLE_SELECT(1007, "Single-Select (Drop down)"),
+                MULTI_SELECT(1013, "Multi-Select (List Box)"),
+                READONLY_SINGLE_SELECT_SAVE(1024, "Readonly single-select save"),
+                READONLY_MULTI_SELECT_SAVE(1025, "Readonly multi-select save"),
+                READONLY_SINGLE_SELECT_NO_SAVE(1027, "Readonly single-select no save"),
+                READONLY_MULTI_SELECT_NO_SAVE(1028, "Readonly multi-select no save"),
+                CODE_LOOKUP(1031, "Code Lookup");
+
+                private final long key;
+                private final String value;
+
+                CodedQuestionType(long key, String value) {
+                        this.key = key;
+                        this.value = value;
+                }
```

Comment:
Suggestion: If we change this to `long value, String name` then the UI can use the options directly in the `SelectInput` component without need to map them